docs/content/posts/2025-11-30-release-2.2.0.md

```diff
@@ -0,0 +1,312 @@
+---
+title: "Apache Flink 2.2.0: Advancing Real-Time Data + AI and Empowering Stream Processing for the AI Era"
+date: "2025-11-30T00:00:00.000Z"
```

Comment:
TODO：plz update all date and date in url

Reply:
Change to 12.03.